### ISSUE TITLE
 Crown of the Skies in groups

### ISSUE THREAD

**Issue by Waisse** [Divine-Journey-2/main#205]
**Modpack version:**
2.1.0

**Describe the request:**
The Crown of the Skies is required to enter the Aether, but the cost to make more than one is way too high in a group so multiple players can also get into the dimension. A way to dupe it would be nice.

**Reasons why this should be added:**
To make playing in a group less annoying. The Crown is consumed to allow ONE player access to the Aether.

**Describe alternatives you've considered:**
Crown dupe recipe, or let players get inside the Aether without consuming a Crown.

**Additional context:**
There is none, this is exclusively an issue for groups who wants to tackle the Aether without abusing tp commands to all get inside the dimension. We just had the based admin cheat one for each of us but I'm still making this issue because not every admin is that based. Have a nice day.

**Comment by Atricos** [comment]
The Crown of Skies can now be duplicated with 2 Hardened Amber Mooshes, and added a tooltip for it to let people know what the duplication recipe is for.

Fix available in version 2.2.0:
https://www.curseforge.com/minecraft/modpacks/divine-journey-2/files/3449256

**Comment by Atricos** [comment]
I realized I managed to mess up the recipe. It currently is:
```
recipes.addShaped(<contenttweaker:crown_of_skies>,
[[<contenttweaker:hardened_amber_moosh>,<contenttweaker:crown_of_skies>,<contenttweaker:hardened_amber_moosh>]]);
```
But it should be:
```
recipes.addShaped(<contenttweaker:crown_of_skies>,
[[<contenttweaker:hardened_amber_moosh>,<contenttweaker:crown_of_skies>.reuse(),<contenttweaker:hardened_amber_moosh>]]);
```

I'll fix this next update then.

**Comment by Atricos** [comment]
Implemented the change into 2.3.0.
